double_square.cc

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+    int n; 
+    double t1 = 2, nextTerm = 0;
```

Comment:
This is rather complex. ideally you want to return your variable in every "loop". There is no need to implement the loop in C++. In turn you could check if the return still fit into the Datatype, which you don't do. You basically check the output if it makes sense to you, not the computer. So you depend somewhat on what happens when the Data Type overflows.